superbench/executor/executor.py

```diff
@@ -106,17 +110,47 @@ def __exec_benchmark(self, context, log_suffix):
                     logger.info('Executor succeeded in %s.', log_suffix)
                 else:
                     logger.error('Executor failed in %s.', log_suffix)
+                return benchmark.result
```

Comment:
benchmark.result is only the summary data, we'd better return all info of benchmark such as raw_data and meta_data (start/end time, return code, benchmark name etc.). How about:
`return json.loads(benchmark.serialized_result)`

Reply:
updated